pykernels/graph/randomwalk.py

```diff
@@ -8,7 +8,7 @@
 from pykernels.base import Kernel, GraphKernel
```

Comment:
If we decide to use relative imports, I would keep it consistent, i.e. also change pykernels.base import to .base